docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM php:7.0-fpm-alpine
```

Comment:
if this is still a problem, and if we only care about varnish 5 here for now we can use Debian scratch which is very soon final anyway, and contains apt packages for `varnish` and `varnish-modules`. See: https://github.com/ezsystems/ezplatform/blob/a5711195af5b7f2bfd78d9de21ca62ad304a5fe5/doc/docker-compose/Dockerfile-varnish